pkg/kapp/cmd/app/prev_app_flag.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2022 VMware, Inc.
```

Comment:
I'm not 100% on this name, I wanted to keep it separate from `AppFlags` because that's used more widely than where this feature is useful. But I struggled to come up with something better.

Reply:
It looks good to me.